### ISSUE TITLE
(W3C) WAI

### ISSUE THREAD

**Issue by remibetin** [w3c/translation-glossaries#24]
## Context

As discussed here : https://github.com/w3c/wai-mobile/pull/25#discussion_r600736283

In some pages, reference is made to "W3C WAI".

For example :
- "To learn more about how W3C WAI develops material [...]" [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/)
- "Mobile accessibility is covered in existing W3C WAI accessibility standards/guidelines" / "W3C WAI Addresses Mobile Accessibility" [Mobile Accessibility at W3C](https://www.w3.org/WAI/standards-guidelines/mobile/)
- "Current work on cognitive accessibility at W3C WAI is mostly focused in the following Task Forces" [Cognitive Accessibility at W3C](https://www.w3.org/WAI/cognitive/)

## Issue

In French, we need to switch the terms. W3C WAI is "l'Initiative pour l’accessibilité du Web du W3C".

But in [Introduction à l'accessibilité du web](https://www.w3.org/WAI/fundamentals/accessibility-intro/fr), I noticed there is no switch : "Pour en savoir plus sur la façon dont W3C WAI développe des documents" (ping @Sylvie42)

In https://github.com/w3c/wai-mobile/pull/25, @Alex-akel uses the same form "L'accessibilité mobile est traitée dans les standards/règlements existant du W3C WAI"

I think it needs improvement and it is not easy to understand. In French, the WAI is the first complement and "initiative" is feminine so at the very least, we have to write "de la WAI du W3C". But is not great too.

That is why I would suggest using the same principle as in the [spanish translation](https://www.w3.org/WAI/standards-guidelines/mobile/es) : "l'Initiative pour l'accessibilité du Web (WAI) du W3C"

## Suggestions

For "W3C WAI" :
- l'Initiative pour l'accessibilité du Web (WAI) du W3C (my personal pick)
- la WAI du W3C

For "WAI", I suggest using "la WAI" to be consistent with the authorized [WCAG 2.0 translation](https://www.w3.org/Translations/WCAG20-fr/)

What do you think ?

**Comment by shawna-slh** [comment]
The [WCAG 2.0 Authorized Translation](https://www.w3.org/Translations/WCAG20-fr/ ) is well vetted and authoritative. When a word or phrase is in that document, please always use it.

**Comment by remibetin** [comment]
> The [WCAG 2.0 Authorized Translation](https://www.w3.org/Translations/WCAG20-fr/) is well vetted and authoritative. When a word or phrase is in that document, please always use it.

Indeed ! But there is no occurrence of « W3C WAI » in WCAG 2.0 😢

**Comment by shawna-slh** [comment]
hummm. [General glossary](https://github.com/w3c/translation-glossaries/blob/main/general.md#acronyms-w3c-wai-html-wcag-aria-eowg-etc) says "Leave the acronym in English."

So if it's actually, "W3C WAI", then I guess it should stay "W3C WAI"?
Or, do you propose a change to the general glossary, too?

Or, are you wanting to figure out places where we use "World Wide Web Consortium (W3C) Web Accessibility Initiative (WAI)"?

If the latter, let's get input from Sylvie and others...

**Comment by shawna-slh** [comment]
I looked at some W3C press releases translated into French, and it is done inconsistently.

For example, "World Wide Web Consortium (W3C)" is left in English in the "boilerplate" text at the bottom of https://www.w3.org/2021/01/pressrelease-webrtc-rec.html.fr

However, it is translated in this one:
* [Le W3C lance un cours en ligne gratuit « introduction à l'accessibilité Web » ...](https://www.w3.org/2019/12/pressrelease-intro-web-accessibility-course.html.fr)
* [W3C launches an Introduction to Web Accessibility free online course...](https://www.w3.org/2019/12/pressrelease-intro-web-accessibility-course.html.en)

Let's see if any W3C Comm folks (some French speaking) and/or Internationalization folks can help us reach a W3C-wide decision for translating into French (or leaving in English) :
* World Wide Web Consortium (W3C) Web Accessibility Initiative (WAI)
* W3C Web Accessibility Initiative (WAI)

**Comment by koalie** [comment]
Hello @shawna-slh and @remibetin 

My strong preference is to treat "W3C WAI" as a brand, therefore as a proper noun. It means that in gender-based languages no articles are used. 
E.g., "W3C WAI technical activities" becomes "les activités techniques de W3C WAI". Or "Refer to WAI materials" becomes "Voir les ressources WAI".

I suppose that the first occurrence in French could be "L'Initiative pour l’accessibilité du Web du [or 'de'] W3C" as @remibetin suggested, unless using "W3C WAI" is fine.

To respond to Shawn's list, I would say that in French, all of the 3 would very likely be translated the same way: "L'Initiative pour l’accessibilité du Web du [or 'de'] W3C".

**Comment by shawna-slh** [comment]
@remibetin , as koalie is Head of W3C Communications and a native French speaker, so let's go with her strong preference! Would you update the **glossary** with correct translation of the various versions of the phrases used in WAI resources. Thanks!

[side note : for the many of the WAI resources, we consider "W3C" more recognizable than  "World Wide Web Consortium" and do not write out "W3C". We can revisit that where suggested. The translation should use the same treatment of written out or just acronym as the English does.]

**Comment by remibetin** [comment]
Hi @shawna-slh & @koalie 

Thanks for the input !

> My strong preference is to treat "W3C WAI" as a brand, therefore as a proper noun. It means that in gender-based languages no articles are used. E.g., "W3C WAI technical activities" becomes "les activités techniques de W3C WAI". 

OK to treat "W3C WAI" as a brand.

> Or "Refer to WAI materials" becomes "Voir les ressources WAI".

For "WAI" used without W3C, are we sure we want a translation different from the authorized WCAG translation ? 
Examples from WCAG 2.0 : "l'activité technique de **la** WAI", "ressources de **la** WAI"

Here are my suggestions to reach a consensus ?
* We use an article when "W3C" is used alone
  *  "Le W3C recommande" (WCAG 2.0), 
  * "Recommandation du W3C" (WCAG 2.0),
  * "document du W3C" (WCAG 2.0)
* We use an article when "WAI" is used alone
  * "l'activité technique de la WAI" (WCAG 2.0), 
  * "les ressources de la WAI" (WCAG 2.0), 
  * "WAI Resources" (in [Using WAI Material](https://www.w3.org/WAI/about/using-wai-material/)) becomes "ressources de la WAI".
* We treat "W3C WAI" as a brand, without articles
  * "les standards/règlements existants de W3C WAI", 
  * "Pour en savoir plus sur la façon dont W3C WAI développe des documents".
* When WAI is written out, we translate it fully in French
  *  "The Web Accessibility Initiative (WAI) is an initiative of the W3C"  (in [About W3C WAI](https://www.w3.org/WAI/about/)) becomes "L'Initiative pour l'accessibilité du Web (WAI) est une initiative du W3C",
  * "The W3C Web Accessibility Initiative (WAI)" (in [Participating in WAI](https://www.w3.org/WAI/about/participating/)) becomes "L'Initiative pour l'accessibilité du Web (WAI) du W3C".

**Comment by Sylvie42** [comment]
Hello,
I think those suggestions are very good.
Best
Sylvie
De : Rémi Bétin ***@***.***>
Envoyé : mercredi 28 avril 2021 20:32
À : w3c/translation-glossaries ***@***.***>
Cc : Sylvie Duchateau ***@***.***>; Mention ***@***.***>
Objet : Re: [w3c/translation-glossaries] W3C WAI (#24)


Hi @shawna-slh<https://github.com/shawna-slh> & @koalie<https://github.com/koalie>

Thanks for the input !

My strong preference is to treat "W3C WAI" as a brand, therefore as a proper noun. It means that in gender-based languages no articles are used. E.g., "W3C WAI technical activities" becomes "les activités techniques de W3C WAI".

OK to treat "W3C WAI" as a brand.

Or "Refer to WAI materials" becomes "Voir les ressources WAI".

For "WAI" used without W3C, are we sure we want a translation different from the authorized WCAG translation ?
Examples from WCAG 2.0 : "l'activité technique de la WAI", "ressources de la WAI"

Here are my suggestions to reach a consensus ?

  *   We use an article when "W3C" is used alone
     *   "Le W3C recommande" (WCAG 2.0),
     *   "Recommandation du W3C" (WCAG 2.0),
     *   "document du W3C" (WCAG 2.0)
  *   We use an article when "WAI" is used alone
     *   "l'activité technique de la WAI" (WCAG 2.0),
     *   "les ressources de la WAI" (WCAG 2.0),
     *   "WAI Resources" (in Using WAI Material<https://www.w3.org/WAI/about/using-wai-material/>) becomes "ressources de la WAI".
  *   We treat "W3C WAI" as a brand, without articles
     *   "les standards/règlements existants de W3C WAI",
     *   "Pour en savoir plus sur la façon dont W3C WAI développe des documents".
  *   When WAI is written out, we translate it fully in French
     *   "The Web Accessibility Initiative (WAI) is an initiative of the W3C" (in About W3C WAI<https://www.w3.org/WAI/about/>) becomes "L'Initiative pour l'accessibilité du Web (WAI) est une initiative du W3C",
     *   "The W3C Web Accessibility Initiative (WAI)" (in Participating in WAI<https://www.w3.org/WAI/about/participating/>) becomes "L'Initiative pour l'accessibilité du Web (WAI) du W3C".

**Comment by koalie** [comment]
Hello @remibetin 

> For "WAI" used without W3C, are we sure we want a translation different from the authorized WCAG translation ?

That would indeed be my preference.
I recognize the discrepancy it would create with the existing French translation of WCAG.

We also plan to document this year our preference to treat "W3C" alone as a brand and I do recognize it may take a while for it to be adopted. Part of our documentation will include continuing using the acronym W3C when convenient but to start using "the Web Consortium" when possible. 

> Examples from WCAG 2.0 : "l'activité technique de **la** WAI", "ressources de **la** WAI"

-1
-> "l'activité technique de WAI" (ou "l'activité technique WAI"), "ressources de WAI" (ou "ressources WAI").

> Here are my suggestions to reach a consensus ?
> 
>     * We use an article when "W3C" is used alone

-1
I propose for cases when the lack of article in French is weitd to swap "W3C" with "Consortium Web"

>       * "Le W3C recommande" (WCAG 2.0),

-> "W3C recommande WCAG 2.0" (or "Le Consortium Web recommande WCAG 2.0")

>       * "Recommandation du W3C" (WCAG 2.0),

-> "Recommandation W3C" (WCAG 2.0)

>       * "document du W3C" (WCAG 2.0)

-> "document W3C (WCAG 2.0)" (or "document du Consortium Web (WCAG 2.0)")


>     * We use an article when "WAI" is used alone

-1

>       * "l'activité technique de la WAI" (WCAG 2.0),

->  "l'activité technique WAI" (ou "l'activité technique de WAI")

>       * "les ressources de la WAI" (WCAG 2.0),

-> "les ressources WAI" (ou "les ressources de WAI")


>     * We treat "W3C WAI" as a brand, without articles
>       
>       * "les standards/règlements existants de W3C WAI",
>       * "Pour en savoir plus sur la façon dont W3C WAI développe des documents".

+1

> 
>     * When WAI is written out, we translate it fully in French
>       
>       * "The Web Accessibility Initiative (WAI) is an initiative of the W3C"  (in [About W3C WAI](https://www.w3.org/WAI/about/)) becomes "L'Initiative pour l'accessibilité du Web (WAI) est une initiative du W3C",

-> "L'Initiative pour l'accessibilité du Web (WAI) est une initiative W3C" (or "L'Initiative pour l'accessibilité du Web (WAI) est une initiative du Consortium Web")

>       * "The W3C Web Accessibility Initiative (WAI)" (in [Participating in WAI](https://www.w3.org/WAI/about/participating/)) becomes "L'Initiative pour l'accessibilité du Web (WAI) du W3C".

-> "L'Initiative pour l'accessibilité du Web (WAI) du Consortium Web"

**Comment by remibetin** [comment]
@koalie Thank you so much for your feeback ! It helps understand your brand strategy and of course we have to stick with it.

@shawna-slh Now that the issue is clarified, I'll update the glossary.

**Comment by shawna-slh** [comment]
Thanks @remibetin and koalie !

The WCAG 2.1 French Authorized Translation work is starting up again, and hope to finish next month. So we can get it correct in that. (Remi, as soon as one thing is settled, I planned to invite you and others to contribute to that if you want.)

Remi, once you update the Glossary, let's point current French translators to it.

Coralie, I'll send e-mail to Comm team about any new guidance for translations.
